community/server/src/docs/ops/powershell.asciidoc

```diff
@@ -44,11 +33,29 @@ Set-ExecutionPolicy -ExecutionPolicy RemoteSigned
 For more information see http://go.microsoft.com/fwlink/?LinkID=135[About execution policies].
 --
 
+[NOTE]

```

Comment:
This section now consists entirely _NOTE_ admonitions. Notes, tips, warnings, etc., are meant to highlight something within the ordinary flow of the text. There is no ordinary text flow in this paragraph, so it defeats the purpose to use notes. It would be better to just turn all three notes into ordinary paragraphs.